rust-symcrypt/src/cipher/mod.rs

```diff
@@ -0,0 +1,192 @@
+//! Block Cipher functions related to creating expanded keys. For further information please see symcrypt.h for more info
```

Comment:
why is this file called mod.rs?

Reply:
Its just a way to separate your code into folders so the `mod.rs` under `ciphers/mod.rs` will group all of the cipher related functions, similar to how we have `ecc/mod.rs` or `rsa/mod.rs`, it sort of acts like a mini library,

So when you call this code from an external crate or another part of the code base you would do `cipher::aes_expand_key` , or `rsa::generate_new_key_pair()`

Reply:
@samuel-lee-msft It's mod as in module, not modulus :)